### PR TITLE
Monitor jobs and logs

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: ocrd_lib.sh [hunk 1]
```diff
@@ -170,6 +178,6 @@ close() {
     # become synchronous again
     logger -p user.info -t $TASK "ocr_exit in sync mode - wait until the processing is completed"
     wait $!
-    rm -f /run/lock/ocr.pid/$PID
+    #rm -f /run/lock/ocrd.jobs/$REMOTEDIR
```
Comment (markusweigelt) [inline ocrd_lib.sh:181]: If it is uncommented then we can remove this or is it necessary?

File: ocrd_monitor/docker-compose.yml [hunk 2]
```diff
@@ -12,9 +12,28 @@ services:
...
+  dozzle:
+    image: amir20/dozzle:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - ${MONITOR_PORT_LOG}:8080
+    environment:
+      - DOZZLE_FILTER=name=kitodo_production_ocrd
+      # DOZZLE_USERNAME=
+      # DOZZLE_PASSWORD=
```
Comment (markusweigelt) [inline ocrd_monitor/docker-compose.yml:36]: Are these environment variables necessary. Are these environment variables necessary. If not can they be removed?

File: ocrd_monitor/docker-compose.yml [hunk 3]
```diff
@@ -12,9 +12,28 @@ services:
...
 
     volumes:
       - ${MONITOR_DATA}:/data
+      - ${MANAGER_KEY}:/id_rsa
+      - shared:/run/lock/ocrd.jobs
+
+  dozzle:
+    image: amir20/dozzle:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```
Comment (markusweigelt) [inline ocrd_monitor/docker-compose.yml:30]: Is this necessary? If yes we should add a env variable for other environments of host system.

File: ocrd_monitor/docker-compose.yml (hunk 2 above)
Comment (bertsky) [reply ocrd_monitor/docker-compose.yml:36]: It's illustrative of what we could do for a simple authentication.

File: ocrd_monitor/docker-compose.yml (hunk 3 above)
Comment (bertsky) [reply ocrd_monitor/docker-compose.yml:30]: Yes, this is essential. It gives the container access to the Docker daemon.

If non-Unix hosts use a different location, then we must try to find a more general formulation.

Comment (markusweigelt) [reply ocrd_monitor/docker-compose.yml:30]: Yes in my case it is not possible.

```
Error response from daemon: _modules\ocrd_manager\ocrd_monitor\var\run\docker.sock%!(EXTRA string=is not a valid Windows path)
make: *** [Makefile:62: start] Error 1

```

Comment (markusweigelt) [reply ocrd_monitor/docker-compose.yml:30]: Adding another slash as first character of host path seems to work on windows environments.

https://stackoverflow.com/questions/36765138/bind-to-docker-socket-on-windows/58204900#58204900

Comment (bertsky) [reply ocrd_monitor/docker-compose.yml:30]: Feel free to modify the branch!

**Comment by markusweigelt** [comment]
I added a env with default to docker compose and rename the port envs to more contextual scheme.

Do you think it is good to rename dozzle envs to DOZZLE instead of MONITOR? Atm the prefix of our remaining envs is service specific.

**Comment by bertsky** [comment]
> I added a env with default to docker compose and rename the port envs to more contextual scheme.
> 
> Do you think it is good to rename dozzle envs to DOZZLE instead of MONITOR? Atm the prefix of our remaining envs is service specific.

Wait.

Why introduce an additional env var which the user likely will not comprehend when simply using double-slash would suffice on both host systems? 

Also, I liked the port mnemonics just fine (it's your `CONTROLLER_SSH_PORT` which first deviated from the scheme).

**Pull request merged**